src/deadline/client/cli/_groups/bundle_group.py

```diff
@@ -82,6 +82,7 @@ def validate_parameters(ctx, param, value):
 @click.option("--profile", help="The AWS profile to use.")
 @click.option("--farm-id", help="The farm to use.")
 @click.option("--queue-id", help="The queue to use.")
+@click.option("--storage-profile-id", help="The storage profile to use.")
```

Comment:
gui-submit implementation is being punted until they share options?

Reply:
Good point. I suppose I *should* add it to there as well...

Reply:
Update... I took a look through the implementation.

Implementing it for `gui-submit` would change the config file setting 'cause gui-submit doesn't pass around a config object. It relies on the singleton nature of the config. So, I think that implementing the option for gui-submit would not be great in that the options would behave differently. `bundle submit` would treat it as a temporary override for just this submission, but `bundle gui-submit` would treat it as doing a `config set` beforehand. 

So, I'm inclined to leave this as-is. What do you think?